.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,36 @@
+# Summary of Changes
+- Include a summary of changes made by this pull request
```

Comment:
Should this be marked as text to be replaced (e.g. with _emphasis_)?

Reply:
Text has been emphasized. I've also added opening and closing angle brackets around it.